src/ethereum/fees/ethMiningFees.ts

```diff
@@ -206,7 +206,9 @@ const MAX_SIGNATURE_COST = '1040' // (32 + 32 + 1) * 16 max cost for adding r, s
 // This is a naive (optimistic??) implementation but is good enough as an
 // estimate since it isn't possible to calculate this exactly without having
 // signatures yet.
-export const calcL1RollupFees = (params: CalcL1RollupFeeParams): string => {
+export const calcOptimismL1RollupFees = (
```

Comment:
Should this be called `calcOptimismRollupFees`?

Edit: I see you keep the "L1" part in the Arbitum types too, so it seems like you probably have a reason for this to be different.